common/lib/xmodule/xmodule/template_module.py

```diff
@@ -20,8 +20,10 @@
 )
 from xmodule.xml_module import XmlMixin
 
+from openedx.core.djangolib.markup import Text
 
-class CustomTagBlock(
+
+class CustomTagTemplateBlock(  # pylint: disable=abstract-method
```

Comment:
`CustomTagDescriptor` [inherited](https://github.com/edx/edx-platform/pull/26873/files#diff-8fd8bc5ed1b612e693dd31c00bb8b91fb3b808c37ee3be73217b5c8f0f997f0cL39) from `RawDescriptor` and `custom_tag_template` pointed to `RawDescriptor` as well. So I have made this the base class and `CustomTagBlock` inherited from it.